src/reducers/auth.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import type {Action} from '../actions/types';
+
+type AuthState = {
+  state: 'mystery' | 'logged in' | 'logged out' | 'login failed',
```

Comment:
this is a "union type" in flow, it basically says that the `state` value here can take one of four values and _nothing else._ if you remove 'mystery' and then run flow, it should complain about the default state on line 15.